src/deploy/osp_deployer/director.py

```diff
@@ -1239,8 +1242,8 @@ def run_sanity_test(self):
             cmd = 'rm -f ~/{}'.format(self.settings.sanity_key_name)
             self.run_tty(cmd)
             self.run_tty('wget '
-                         'http://download.cirros-cloud.net/0.3.3/'
-                         'cirros-0.3.3-x86_64-disk.img')
+                         'http://cloud.centos.org/centos/7/images/'
+                         'CentOS-7-x86_64-GenericCloud.qcow2')
             self.run_tty("cd " + self.validation_dir +
```

Comment:
Image is being downloaded during test already, i think we can get rid of the wget above